integrations/sbt-bloop/src/main/scala/bloop/integrations/sbt/SbtBloop.scala

```diff
@@ -625,7 +623,7 @@ object BloopDefaults {
             case (options, sbt.Tests.Exclude(tests)) =>
               options.copy(excludes = tests.toList ++ options.excludes)
             case (options, other: sbt.TestOption) =>
-              logger.info(s"Skipped test option '${other}' as it can only be used within sbt.")
```

Comment:
FTR, i find it better to wrap them in curly braces so that you don't need to add them whenever expressions are added.

Reply:
It is a matter of taste. I prefer the more concise notation and this is also the style IntelliJ recommends.

Reply:
Let's keep these changes as they are, but let's not do more of these stylistic subjective changes in the future. We have too many things to improve in the codebase and we don't wanna waste time discussing about syntax or coding style :wink:

Reply:
Agreed. If we all agree on using a particular code linter configuration (such as IntelliJ's defaults), we could avoid these discussions.

Reply:
I don't use such defaults, so it's better that everyone makes the necessary changes with their own style, only touching the code required for the change. That helps keep git history clean.